src/pages/auth/ChoosingUser.js

```diff
@@ -1,68 +1,72 @@
-
 /**
  * @summary This file renders the Choose user type component
  * Main component: LoginPage
  * @author Ken Pham
  */
 
-import React from 'react';
-import FeedbackImage from '../../components/FeedbackImage';
+import React from "react";
+import FeedbackImage from "../../components/FeedbackImage";
 
 // Constants
-import * as userTypes from '../../constants/user-types'
+import * as userTypes from "../../constants/user-types";
 
 // Styles
-import './styles/ChoosingUser.css'
+import "./styles/ChoosingUser.css";
 
 /**
  * @summary Rendering the page which users use to choose their user type
- * @param {} props 
+ * @param {} props
  * @returns JSX Elements
  */
-const ChoosingUser = props => {
-
-    /**
-     * @summary Navigate to the Login Page with the user type of SPU
-     */
-    const _onChooseStudentRole = () => {
-        props.history.push('/login',{
-            userType: userTypes.STUDENT
-        });
-    }
+const ChoosingUser = (props) => {
+  /**
+   * @summary Navigate to the Login Page with the user type of SPU
+   */
```

Comment:
These comments must be put above the function declaration
```suggestion
  /**
   * @summary Navigate to the Login Page with the user type of SPU
   */
const ChoosingUser = (props) => {
```

Reply:
Not sure why this happened since I don't remember touching this part, but will also fix that.